gps/app/views/onboarding/Onboarding1.js

```diff
@@ -99,7 +99,7 @@ class Onboarding extends Component {
               <View style={sharedStyles.footerContainer}>
                 <EulaModal
                   continueFunction={() =>
-                    this.props.navigation.replace('Onboarding2')
```

Comment:
Best practices on preserving navigation stack. Lmk if this was intentional.

Reply:
I think it was intentional because it's an overlay on the first screen. If the back button presses dont feel wonky, its ok to change.

Reply:
👍 that's a good reason. Will keep